scripts/helper/functions.sh

```diff
@@ -52,7 +52,7 @@ preflight_checks()
 
 get_kafka_cluster_id_from_container()
 {
-  KAFKA_CLUSTER_ID=$(curl -s http://kafka1:8091/v1/metadata/id | jq -r ".id")
+  KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/kafka/v3/clusters | jq ".data[0].cluster_id")
```

Comment:
This breaks the demo with the following failure:

```
[8:40:34] ~/git/cp-demo(PR-239) ✗: curl -s http://localhost:8091/kafka/v3/clusters 
<html>
<head>
<meta http-equiv="Content-Type" content="text/html;charset=utf-8"/>
<title>Error 401 Unauthorized</title>
</head>
<body><h2>HTTP ERROR 401 Unauthorized</h2>
<table>
<tr><th>URI:</th><td>/kafka/v3/clusters</td></tr>
<tr><th>STATUS:</th><td>401</td></tr>
<tr><th>MESSAGE:</th><td>Unauthorized</td></tr>
<tr><th>SERVLET:</th><td>default</td></tr>
</table>

</body>
</html>
```

The original works:

```
[8:40:32] ~/git/cp-demo(PR-239) ✗: curl -s http://localhost:8091/v1/metadata/id    
{"id":"QTtyaXufRuOfIUCj2JIcLw","scope":{"path":[],"clusters":{"kafka-cluster":"QTtyaXufRuOfIUCj2JIcLw"}}}  
```

For all instances (code & docs) where the Kafka cluster ID is determined, can we restore it to the original?

Reply:
hm that is odd because it certainly worked before. I'll back it out for now and investigate separately. May be a regression that needs to be addressed in 6.0.1.